src/pagingscrollview/src/NIPagingScrollView.m

```diff
@@ -418,6 +421,26 @@ - (void)layoutVisiblePages {
   }
 }
 
+- (void)dropPage:(UIView<NIPagingScrollViewPage> *)page {
```

Comment:
The call sites of this method (e.g., recyclePageAtIndex:) and the invocation of didRecyclePage: might suggest naming this method recyclePage:?